essential-projects/datasource_adapter_postgres
# 🐛 Fix entityTypeKey usage

## What did you change?

The regular skeleton is broken, because at some points the Entity-suffix is used where it shouldn't be used and vice versa. this hopefully fixes this usage, at least for now

## How can others test the changes?

check out the skeleton, go to the regular server, install the dependencies and try staring it, which will fail. Then apply the changes, and try again

## PR-Checklist

Please check the boxes in this list after submitting your PR:
